neo.UnitTests/IO/Caching/UT_Cache.cs

```diff
@@ -57,18 +57,12 @@ public IEnumerator MyGetEnumerator()
     [TestClass]
     public class UT_Cache
     {
-        MyCache cache;
         readonly int max_capacity = 4;
 
-        [TestInitialize]
-        public void init()
-        {
-            cache = new MyCache(max_capacity);
-        }
-
         [TestMethod]
         public void TestCount()
         {
+            var cache = new MyCache(max_capacity);
```

Comment:
Why this change? Can't we have this in the test initializer?
https://nunit.org/docs/2.2.7/setup.html